Add first task

mgandore opened the pull request.

mgandore committed:
Add first task

danielursu commented on tasks/.editorconfig:
```diff
@@ -0,0 +1,16 @@
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
```
```suggestion
indent_style = tab
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true

[*.ts]
quote_type = double
```

danielursu replied:
let's change this in order to have a proper convention

danielursu commented on tasks/src/app/app.component.ts:
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
```
format identation to tabs 4

danielursu replied:
please format all the component files

danielursu commented on tasks/src/app/custom-button/custom-button.component.html:
```diff
@@ -0,0 +1,2 @@
+<p>custom-button works!</p>
```
delete unnecessary line

danielursu commented on tasks/src/app/app.component.ts:
```diff
@@ -0,0 +1,10 @@
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'tasks';
```
don't need anymore
```suggestion
```

mgandore committed:
Change .editorconfig according to the convention

danielursu commented on tasks/src/app/custom-button/custom-button.component.ts:
```diff
@@ -0,0 +1,15 @@
+})
+export class CustomButtonComponent implements OnInit {
+
+  constructor() { }
```
```suggestion
  public constructor() { }
```

danielursu replied:
let's add access modifiers

danielursu commented on tasks/src/app/custom-button/custom-button.component.ts:
```diff
@@ -0,0 +1,15 @@
+
+  constructor() { }
+
+  ngOnInit(): void {
```
```suggestion
  public ngOnInit(): void {
```

mgandore committed:
Make requested changes.

Added the word 'public' to the public methods to improve code's
readability.
Deleted unnecessary :html element in custom-button component and
title of AppComponent class.
